autoload/tagbar.vim

```diff
@@ -3162,7 +3163,11 @@ function! s:ExecuteCtags(ctags_cmd) abort
         call tagbar#debug#log('Exit code: ' . v:shell_error)
         redraw!
     else
-        let py_version = get(g:, 'tagbar_python', 1)
+        if has('win32')
+          let py_version = get(g:, 'tagbar_python', 0)
+        else
+          let py_version = get(g:, 'tagbar_python', 1)
+        endif
```

Comment:
Unfortunately this change doesn't make a whole lot of sense. Inside the `s:run_system()` function call, there is a check already for `if has('win32')` and is specifically going into the python handler for that. The original issue was raised specifically against gvim, not just vim. So we need to understand if this is required for all windows installs, or if this is specific to only gvim installations on windows. If we can get some confirmation if this is only for gvim, then we would want to handle this in the `s:run_system()` function adding a `!has('gui_running')` check:

```vim
function! s:run_system(cmd, version) abort
    if has('win32') && !has('nvim') && !has('gui_running') && a:version > 0 && (has('python3') || has('python2'))
```

Otherwise if this does effect every windows installation, then still it would be better to handle it in the `s:run_system()` function call adding a `!` in front of the `has('win32')` check:
```vim
function! s:run_system(cmd, version) abort
    if !has('win32') && !has('nvim') && a:version > 0 && (has('python3') || has('python2'))
```

So if someone can help confirm the correct behavior on windows for both GUI and non-GUI installations, then we should update it like one of these options.